lib/bamboozled.rb

```diff
@@ -8,7 +8,7 @@
 require_relative './bamboozled/errors'
 require_relative './bamboozled/base'
 
-%w(base employee report time_off meta).each {|a| require_relative "./bamboozled/api/#{a}"}
+%w(base employee report time_off meta tabular).each {|a| require_relative "./bamboozled/api/#{a}"}
```

Comment:
Line is too long. [98/80]<br>Space between { and | missing.<br>Space missing inside }.